tests/integration/utils.js

```diff
@@ -20,7 +20,37 @@ function insertUser (values) {
   return db.query(insertUser)
 }
 
+async function getUser (userId) {
+  const getUser = {
+    text: 'SELECT * FROM users WHERE id = $1',
+    values: [userId]
+
+  }
+  const user = await db.query(getUser)
+  return user.rows[0]
+}
```

Comment:
Why can't we just use `UserModel` functions? We're just duplicating the code

Reply:
Some of the functions have different inputs. We don't seem to have a `getUser` by userId. And the `checkFriendshipExists` takes email which we don't have.

Reply:
We should then add it to the `UserModel` then. Do you feel like the file is getting too big / unwieldy?

Reply:
They dont really see any use outside of testing which is why I made some functions just so I didnt have to specify more constants in the tests. The file is fine right now but we should split it up at some point.